.github/workflows/unit-test-pr.yml

```diff
@@ -19,12 +19,23 @@ jobs:
       - name: Cache 📦
         uses: actions/setup-node@v2
         with:
-          node-version: '14'
+          node-version: '16'
           cache: 'pnpm'
 
-      - name: Install and Build 🔧
-        run: |
-          pnpm install
-          pnpm run lint
-          pnpm run test:unit
-          pnpm run test:e2e
+      - name: Install dependencies 🔧
+        run: pnpm install
+
+      - name: Check style and formatting 🧹
+        run: pnpm run lint
+
+      - name: Run unit tests 🧪
+        run: pnpm run test:unit
+
+      - name: Run Cypress E2E tests 🧪
+        uses: cypress-io/github-action@v2
+        with:
+          start: pnpm run dev
```

Comment:
Testing in development environment might be different from the production one. I think, we could change this to build + start commands

Reply:
Good point :+1: